readers-frontend/src/components/SolidJS/MuseNavbar.tsx

```diff
@@ -0,0 +1,85 @@
+import type { MuseNavbarCategory } from "@components/types";
+import type { categories } from "@prisma/client";
+import { Accessor, Component, For, Setter } from "solid-js";
+
+type MuseNavbarProps = {
+  active: Accessor<string>;
+  setActive: Setter<string>;
+  categories: MuseNavbarCategory[];
+};
+
+const MuseNavbar: Component<MuseNavbarProps> = (props) => {
+  return (
+    <>
+      <div
+        class="2xl:text-large hidden h-12 w-full text-base text-white sm:block 2xl:px-[13%]"
+        id="muse-navbar-desktop"
+      >
+        <ul class="flex h-full w-full flex-row justify-between border-t-[1px] border-transparent border-white">
+          <For each={props.categories}>
+            {(category, i) => (
+              <>
+                {i() !== 0 && ( // No divider before Home
+                  <span class="h-3/5 self-center border-r-2 border-white" />
+                )}
+                <li
+                  class={`group relative flex h-full w-full border-b-4 ${
+                    props.active() === category.name
+                      ? "border-white"
+                      : "border-transparent"
+                  }`}
+                >
+                  {props.active() === category.name ? (
+                    <a class="mx-auto my-auto" href={`/${category.name}`}>
```

Comment:
Not sure how but on my end the first muse category (features) is linking to the muse homepage and not that category's page.

Reply:
Yes I made it like that on purpose, I thought the features is their home? Like the last featured articles or whatever

Reply:
That's confusing because "Features" is its own Muse section and not the Home section. If you go onto the admin side, you can see that featured Muse articles are different from featured "Features" articles.

I'd say we should add the Home section like the design to match how articles are currently featured

Reply:
Sure but then I don't think we can fit all the categories in

Reply:
I've tried it manually on my end and it wasn't horrible. But if it looks bad on your end, send me a message